melodic/distribution.yaml

```diff
@@ -10663,17 +10663,17 @@ repositories:
     doc:
       type: git
       url: https://github.com/mgruhler/soem.git
-      version: master
+      version: melodic 
```

Comment:
There's a trailing whitespace here and in the source below.

Reply:
Thanks.